app/components/candidate_interface/course_choices_review_component.rb

```diff
@@ -10,7 +10,8 @@ def initialize(
       show_status: false,
       show_incomplete: false,
       missing_error: false,
-      application_choice_error: false
+      application_choice_error: false,
+      render_link_to_find_when_rejected_on_qualifications: false
```

Comment:
I've been trying to come up with a succinct descriptive name for this.`link_to_find_when_rejected_on_qualifications` is the best I've managed which is still verbose. 🤔